OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -33,12 +33,14 @@ class Repairable : IIssueOrder, IResolveOrder, IOrderVoice
 	{
 		readonly RepairableInfo info;
 		readonly Health health;
+		readonly IMove movement;
 		readonly AmmoPool[] ammoPools;
 
 		public Repairable(Actor self, RepairableInfo info)
 		{
 			this.info = info;
 			health = self.Trait<Health>();
+			movement = self.Trait<IMove>();
```

Comment:
I'll go with `Requires<IMoveInfo>` then, without a trait implementing `IMove` it would later crash anyway.